go.mod

```diff
@@ -380,6 +380,7 @@ require (
 	gopkg.in/gorp.v1 v1.7.2 // indirect
 	gopkg.in/inf.v0 v0.9.1 // indirect
 	gopkg.in/ini.v1 v1.66.2 // indirect
+	gopkg.in/natefinch/lumberjack.v2 v2.0.0
```

Comment:
Oh should we use 2.1?

Reply:
I wonder...

Reply:
There might be a performance benefit to the async rotation :shrug:

Reply:
I'll add the 2.1 then

Reply:
2.1 seems to be updated at the moment for some reason. 2.0.0 has newer commits, so we'll stick with 2.0.0.